File: src/test/java/org/springframework/samples/petclinic/system/CacheConfigurationTests.java
package org.springframework.samples.petclinic.system;

import static org.assertj.core.api.Assertions.assertThat;

import org.junit.jupiter.api.Test;
import org.springframework.boot.cache.autoconfigure.JCacheManagerCustomizer;

class CacheConfigurationTests {

	private final CacheConfiguration configuration = new CacheConfiguration();

	@Test
	void shouldCreateCacheConfigurationCustomizer() {
		JCacheManagerCustomizer customizer = configuration.petclinicCacheConfigurationCustomizer();
		assertThat(customizer).isNotNull();
	}

}

File: src/test/kotlin/org/springframework/samples/petclinic/system/CacheConfigurationTests.kt
package org.springframework.samples.petclinic.system

import org.assertj.core.api.Assertions.assertThat
import org.junit.jupiter.api.Test
import org.springframework.boot.cache.autoconfigure.JCacheManagerCustomizer

class CacheConfigurationTests {

    private val configuration = CacheConfiguration()

    @Test
    fun shouldCreateCacheConfigurationCustomizer() {
        val customizer = configuration.petclinicCacheConfigurationCustomizer()
        assertThat(customizer).isNotNull()
    }

}